rules/autoconsent/pubtech.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "pubtech",
+    "prehideSelectors": ["#pubtech-cmp"],
+    "detectCmp": [
+      { 
+        "exists": "#pubtech-cmp" 
+      }
+    ],
+    "detectPopup": [
+      { 
+        "visible": "#pubtech-cmp" 
+      }
+    ],
+    "optIn": [
+      { 
+        "click": "#pubtech-cmp #pt-accept-all" 
+      }
+    ],
+    "optOut": [
+      {
+        "click": "#pubtech-cmp #pt-close:first-child" 
```

Comment:
This didn't work for me on https://www.lefonti.tv/

Reply:
Now it works - I did not do a conditional branching since I managed to find some common selectors in spite of the autogeneration of most of the selectors